src/networking.c

```diff
@@ -1487,9 +1485,8 @@ void unlinkClient(client *c) {
     /* When client was just unblocked because of a blocking operation,
      * remove it from the list of unblocked clients. */
     if (c->flags & CLIENT_UNBLOCKED) {
-        ln = listSearchKey(server.unblocked_clients,c);
-        serverAssert(ln != NULL);
-        listDelNode(server.unblocked_clients,ln);
+        int result = listSearchDel(server.unblocked_clients,c);
+        serverAssert(result);
```

Comment:
Just an idea of a more descriptive variable name.

```suggestion
        int deleted = listSearchDel(server.unblocked_clients,c);
        serverAssert(deleted);
```

Without variable, just one line `serverAssert(listSearchDel(...))`, would work too.

Reply:
This name could be used in to all places `result` was used on this PR but it's not important.